tools/drprebid/src/main/java/org/prebid/mobile/drprebid/validation/AdServerTest.java

```diff
@@ -8,9 +8,9 @@
 import androidx.annotation.Nullable;
 
 import com.google.android.gms.ads.AdListener;
-import com.google.android.gms.ads.doubleclick.PublisherAdRequest;
-import com.google.android.gms.ads.doubleclick.PublisherAdView;
-import com.google.android.gms.ads.doubleclick.PublisherInterstitialAd;
+import com.google.android.gms.ads.doubleclick.AdManagerAdRequest;
```

Comment:
DrPrebid module has the following issues: 
![Screenshot 2021-04-13 at 14 47 57](https://user-images.githubusercontent.com/79995965/114552557-654f5b80-9c6d-11eb-8664-080c6c0e666c.png)

Reply:
done